client/layout/guided-tours/config-elements/quit.js

```diff
@@ -12,14 +12,16 @@ import { translate } from 'i18n-calypso';
  * Internal dependencies
  */
 import Button from 'components/button';
+import { targetForSlug } from '../positioning';
 import contextTypes from '../context-types';
 
 export default class Quit extends Component {
 	static displayName = 'Quit';
 
 	static propTypes = {
+		click: PropTypes.bool,
 		primary: PropTypes.bool,
-		subtle: PropTypes.bool,
```

Comment:
Was this not used before?

Reply:
Nope.
I think this must be a sort of leftover from `LinkQuit` that, instead, uses a `subtle` prop.